### PR TITLE
Tests about escaping improved

### PR DESCRIPTION
**Pull request by miripiruni**
Fix for #291

### CODE REVIEW

**Comment by miripiruni** [comment]
cc @Yeti-or @zxqfox

File: test/runtime-escaping-test.js [hunk 1]
```diff
@@ -39,12 +39,18 @@ describe('Content escaping', function() {
...
   });
 
-  it('should ignore other props if html exists', function() {
+  it('should ignore html field if block/elem/cls/attrs/tag exists', function() {
     test(function() {
     }, { block: 'b', html: '<unescaped>', content: 'safe text' },
     '<div class="b">safe text</div>');
   });
 
+  it('should render html if only content and html fields exists', function() {
```
Comment (zxqfox) [inline test/runtime-escaping-test.js]: I'd say: `it should determine unescaped html field if there are no bem-entity properties`.

Comment (miripiruni) [inline test/runtime-escaping-test.js]: @zxqfox OK

**Comment by miripiruni** [comment]
Merge?

**Comment by zxqfox** [comment]
👍

**Pull request merged**